keyboards/cospad/keymaps/via/keymap.c

```diff
@@ -0,0 +1,69 @@
+#include QMK_KEYBOARD_H
```

Comment:
Could you add a license header here, please?

Here's a sample:

https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Reply:
added, thank you 😄

Reply:
Can you take a look at this PR?
qmk/qmk_firmware#10531